foundation/foundation-mda/src/main/resources/templates/data-delivery-spark/encryption.java.vm

```diff
@@ -99,15 +99,6 @@ ${step.encryptionSignature} {
                         datasetWithEncryptionPolicyApplied = ds;
                         #end
                         #end
-                        #elseif ($step.hasMessagingInbound())
```

Comment:
Q: Is there a reason for removing this?

Reply:
Yeah.. these were actually not needed and will caused issue since the step with `InboundRecordType` already create the dataset record with schema